src/LeftSidebar.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import VerticalSlider from "./VerticalSlider";
+
+function openLeftSideBar() {
+    document.getElementById("leftSideBar").style.width = "250px";
+}
+
+function closeLeftSidebar() {
+    document.getElementById("leftSideBar").style.width = "0";
+}
+
+const LeftSidebar = () => (
```

Comment:
We can pass in functions as props to allow our sliders to update the size and opacity when their values are changed.

```
      <LeftSidebar setSize={setSize} setOpacity={setOpacity} />
```

then we can access those functions in our sidebar:

```
const LeftSidebar = ({ setSize, setOpacity }) => {
```

and pass down into our `VerticalSlider` as props too
```
           <VerticalSlider
                onChange={handleSizeChange}
                sliderName='BrushSize'
                sliderLabel='Brush Size'
              />
```

then, finally, we can pass that onto our input, so that when it changes the value is updated:

```
const VerticalSlider = ({sliderLabel, sliderName, onChange}) => {
...
<input className='grow rotate-90 w-full h-52 align-middle' onChange={onChange} type="range" min="0" max="100" { ...variableAttribute1 }/>

```